packages/cw721-init-msg-registry/src/contracts/sg721.rs

```diff
@@ -0,0 +1,21 @@
+use cosmwasm_std::{to_binary, Binary, StdResult};
+
+use crate::data::InitMsgData;
+
+/// Generate sg721 init message given ics721 data
+pub fn ics721_get_init_msg(data: &InitMsgData) -> StdResult<Binary> {
+    to_binary(&sg721::InstantiateMsg {
+        name: data.class_id.to_string(),
+        symbol: data.class_id.to_string(),
+        minter: data.ics721_addr.to_string(),
+        collection_info: sg721::CollectionInfo {
```

Comment:
I see that the data is blanked out for now. How would we go about injecting real data into here in the future?